src/cli/one_helper/onezone_helper.rb

```diff
@@ -14,8 +14,373 @@
 # limitations under the License.                                             #
 #--------------------------------------------------------------------------- #
 
+require 'augeas'
```

Comment:
BTW - It would be nice if this is not hard dependency for whole onezone command, but just required for the serversync subcommand. Also, it should fail with some nice error message "ERROR: You need to have Ruby augeas gem installed .... following way ...." if the dependency is not fulfiled.

(Ideally we should also have this new Augeas dependency in opennebula-cli gems, but 99.9% of users don't care about this new serversync feature and require them all to install Augeas devel looks like a big pain to me.